### PR TITLE
Feat: [DSSC#860pvd0b4] Integration de la page de telechargement de formulaire

### PR DESCRIPTION
**Pull request by aertyuiop12345**
Closes #

#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [ ] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [ ] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

File: collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadListActivity.java [hunk 1]
```diff
@@ -179,6 +184,9 @@ private void init(Bundle savedInstanceState) {
             }
         }
 
+        countSelectedItem = findViewById(R.id.count_selected_form);
```
Comment (jgnacadja) [inline collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadListActivity.java:187]: y a t-il un besoin de compter les formulaires sur cette pages ?

File: strings/src/main/res/values/untranslated.xml [hunk 2]
```diff
@@ -35,6 +35,8 @@ the specific language governing permissions and limitations under the License. -
 
     <string name="hash" translatable="false">#</string>
     <string name="hex" translatable="false">Hex</string>
+    <string name="subtitle" translatable="false">SONDAGE</string>
```
Comment (jgnacadja) [inline strings/src/main/res/values/untranslated.xml:38]: ce n'est pas plutôt `enquête`  ?

**Review by jgnacadja — CHANGES_REQUESTED** [review]

File: collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadListActivity.java (hunk 1 above)
Comment (aertyuiop12345) [reply collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadListActivity.java:187]: oui c'est se qui est fait sur la maquette

File: strings/src/main/res/values/untranslated.xml (hunk 2 above)
Comment (aertyuiop12345) [reply strings/src/main/res/values/untranslated.xml:38]: Non c'est Sondage

**Review by jgnacadja — APPROVED** [review]

**Pull request merged**